src/SecTester.Scan/Commands/CreateScan.cs

```diff
@@ -0,0 +1,42 @@
+using System.Net.Http;
+using SecTester.Bus.Commands;
+using SecTester.Core;
+using SecTester.Scan.CI;
+using SecTester.Scan.Content;
+using SecTester.Scan.Models;
+
+namespace SecTester.Scan.Commands;
+
+internal record CreateScan : HttpRequest<Identifiable<string>>
+{
+  public CreateScan(ScanConfig config, HttpContentFactory httpContentFactory, CiDiscovery ciDiscovery,
+    Configuration configuration)
+    : base("/api/v1/scans", HttpMethod.Post)
```

Comment:
The commands should not depend on any kind of service, it makes them unusable. According to the concept, they must be immutable simple POJO/DTO